packages/directory/addon/components/dir-icon.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * {{dir-icon
+ *    value=value
+ * }}
+ */
+import Component from '@ember/component';
+import layout from '../templates/components/dir-icon';
+import { get, computed } from '@ember/object';
+import { A as arr } from '@ember/array';
+import config from 'ember-get-config';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {String} tagName - component should be a span to be on the same line
+   */
+  tagName: 'span',
+
+  /**
+   * @property {Object} fileTypes - types of files in directory route.
+   * default value includes reports and dashboards
```

Comment:
default value jsdoc
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values